engine/src/main/java/org/destinationsol/drawable/GraphicsElement.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.drawable;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.math.Vector2;
+import org.destinationsol.game.drawables.DrawableLevel;
+
+public class GraphicsElement {
+
+    /**
+     * Represents the texture of this graphics element.
```

Comment:
More of a personal preference than a directive or anything, but I like being able to use the one-line approach for documenting fields - at least while the javadoc is fairly short. So that'd simply mean:

`/** Represents the texture of this graphics element. */